src/common/sdk/pnp/web/_add-chain.mdx

```diff
@@ -60,6 +60,10 @@ export type CustomChainConfig = {
    * Number of decimals for the currency ticker (e.g: 18)
    */
   decimals?: number;
+   /**
+   * Logo for the chain
+   */
+  logo: string;
 };
```

Comment:
"isTestnet" is missing in " type CustomChainConfig.
type CustomChainConfig = {"
/**
    Whether the network is testnet or not*/
    isTestnet?: boolean;